system_tests/extnet_tests.py

```diff
@@ -223,22 +198,61 @@ def test_0040_attach_port_group(self):
                 '--port-group', pg_name])
         self.assertEqual(0, result.exit_code)
 
-        def test_0041_detach_port_group(self):
-            """Detach port group from an external network.
-            Invoke the command 'external detach-port-group' in network.
-            """
-            ExtNetTest._client = Environment.get_sys_admin_client()
-            #       platform = Platform(self.client)
-            port_group_helper = PortgroupHelper(ExtNetTest._client)
-            vc_name = self._config['vc2']['vcenter_host_name']
-            pg_name = port_group_helper. \
-                get_ext_net_used_portgroup_name(vc_name, self._name)
-            result = self._runner.invoke(
-                external,
-                args=[
-                    'detach-port-group', self._name, '--vc-name', vc_name,
-                    '--port-group', pg_name])
-            self.assertEqual(0, result.exit_code)
+    def test_0041_detach_port_group(self):
+        """Detach port group from an external network.
+        Invoke the command 'external detach-port-group' in network.
+        """
+        ExtNetTest._client = Environment.get_sys_admin_client()
+        #       platform = Platform(self.client)
+        port_group_helper = PortgroupHelper(ExtNetTest._client)
+        vc_name = self._config['vc2']['vcenter_host_name']
+        pg_name = port_group_helper. \
+            get_ext_net_used_portgroup_name(vc_name, self._name)
+        result = self._runner.invoke(
+            external,
+            args=[
+                'detach-port-group', self._name, '--vc-name', vc_name,
+                '--port-group', pg_name])
+        self.assertEqual(0, result.exit_code)
+
+    def test_0050_add_ip_range(self):
+        """Add an IP range to a subnet in an external network.
+
+        Invoke the command 'external add-ip-range' in network.
+        """
```

Comment:
Any reason "def test_0033_add_ip_range" is moved to test_0050_add_ip_range" ?

Any reason "def test_0031_modify_ip_range(self)" is move to "def test_0051_modify_ip_range(self):" ?

Why  "def test_0052_remove_ip_range(self)" can't be kept as "def test_0034_remove_ip_range(self)" ?

Reply:
It is done for grouping similar test cases like add and enable subnet start from 30 (we have buffer numbers, If any new test cases added in future related to subnet). Similarly ip_range related starting from 50.